plugin/trino-opa/src/test/java/io/trino/plugin/opa/OpaContainer.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.opa;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.lifecycle.Startable;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class OpaContainer
+        implements Startable
+{
+    private static final int OPA_PORT = 8181;
+    private static final String OPA_BASE_PATH = "v1/data/trino/";
+    private static final String OPA_POLICY_PUSH_BASE_PATH = "v1/policies/trino";
+
+    private final GenericContainer<?> container;
+    private URI resolvedUri;
+
+    public OpaContainer()
+    {
+        this.container = new GenericContainer<>(DockerImageName.parse("openpolicyagent/opa:latest-rootless"))
+                .withCommand("run", "--server", "--addr", ":%d".formatted(OPA_PORT))
```

Comment:
Nit: Sorry to bother you with the same again, but we could test we don't fail on the `decision_id` field being present:
```suggestion
                .withCommand("run", "--server", "--addr", ":%d".formatted(OPA_PORT)
                    "--set", "decision_logs.console=true")
```

Reply:
Yup, sorry if I missed this on the other PR. Will add it!